modules/logging-rn-5.7.6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+// logging-5-7-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-6_{context}"]
+= Logging 5.7.6
+This release includes link:https://access.redhat.com/errata/RHSA-2023:4933[OpenShift Logging Bug Fix Release 5.7.6].
+
+[id="openshift-logging-5-7-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
+
+* Before this update, when users pasted a URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
+
+* Before this update, forwarding to Loki using custom labels generated an error when switching from Fluentd to Vector. With this update, the Vector configuration sanitizes labels in the same way as Fluentd to ensure the collector starts and correctly processes labels. (link:https://issues.redhat.com/browse/LOG-4460[LOG-4460])
+
+* Before this update, the Observability Loki console search field did not accept special characters that should escape. With this update, it allows special characters to escape by using the content filter. (link:https://issues.redhat.com/browse/LOG-4456[LOG-4456])
```

Comment:
Not so sure about this one. The console is reachable using a "Logs" link in the UI, but it _is_ only for querying our default instance of Loki, if that's available.

```suggestion
* Before this update, the Observability Logs console search field did not accept special characters that it should escape. With this update, it is escaping special characters properly in the query. (link:https://issues.redhat.com/browse/LOG-4456[LOG-4456])
```